decidim-initiatives/spec/shared/create_initiative_example.rb

```diff
@@ -58,40 +58,6 @@
         end.to change(Decidim::Initiative, :count).by(1)
       end
 
-      context "when attachment is present" do
```

Comment:
I see that's still possible to add attachments to an initiative if the InitiativeType has the "Enable attachments" checkbox active. Why are we removing this? Is there any equivalent spec that I'm missing?